packages/EasyErrorHandler/src/Bridge/Symfony/Translator.php

```diff
@@ -32,10 +31,12 @@ public function __construct(SymfonyTranslatorInterface $decorated, ?string $doma
      */
     public function trans(string $message, array $parameters, ?array $options = null): string
     {
-        return $this->decorated->trans(
-            $message,
-            $parameters,
-            $options[TranslatableExceptionInterface::OPTION_DOMAIN] ?? $this->domain
```

Comment:
we can now remove this constant from the exception interface

Reply:
Noted. Thank you.

Reply:
Removed constant.

Reply:
Thank you, sorry I'm gonna be a pain... we can also remove the `$options` parameter because it was implemented only for that 😝

Reply:
Oh okay got it thanks!